gridt/src/app/core/settings.service.ts

```diff
@@ -68,17 +70,22 @@ export class SettingsService {
   private disabler$ = new BehaviorSubject<boolean>(false);
 
   get isDisabled$ (): Observable<boolean> {
-    //debug
-    return of(false)
-    // return this.disabler$.asObservable();
+    return this.disabler$.asObservable();
+  }
+
+  private Dissable(disabler) {
+    return function(error) {
+      disabler.next(true)
+      return throwError(error);
+    };

```

Comment:
Why is this an observable? I think in this particular case a lot of simplification could be done by simply letting this be a variable, but maybe I do not understand in sufficient depth why you chose to do this. If there is a good reason, a comment would be in order.

Reply:
The idea is to use disabler$ in the html of the settings page to disable certain UI elements. Since the disabler$ may change value while loading the page, I thought it to be wise making it an observable.

Reply:
But maybe you're right, I now see how a variable could be just fine.